packages/alert/index.js

```diff
@@ -10,14 +8,16 @@ import '@warp-ds/icons/elements/warning-16';
 import '@warp-ds/icons/elements/error-16';
 import '@warp-ds/icons/elements/success-16';
 
+import { kebabCaseAttributes } from '../utils';
+
 const variants = {
   negative: 'negative',
   positive: 'positive',
   warning: 'warning',
   info: 'info',
 };
 
-class WarpAlert extends WarpElement {
+class WarpAlert extends kebabCaseAttributes(WarpElement) {
```

Comment:
This shouldn't be necessary in components that have single-word properties/attributes. This wrapper ensures multiword kebab-cased attributes are mapped to camelCase properties. However, if you think it makes sense to  have this wrapper in case a new attribute is added in the future, then I'm fine with that 🙂

Reply:
ah that makes sense! 😄  I just added to it because I thought we wanted to have that for all of our components, but I can remove it! 🙂

Reply:
Updated in this commit: [b7ce216](https://github.com/warp-ds/elements/pull/174/commits/b7ce216ee664d4d87d0c035cd0932d2f840c6255)